tutorials/basics/4_batch.py

```diff
@@ -0,0 +1,283 @@
+"""
+.. currentmodule:: dgl
+
+Batched Graph Classification with DGL
+=====================================
+
+**Author**: `Mufei Li <https://github.com/mufeili>`_,
+`Minjie Wang <https://jermainewang.github.io/>`_
+
+In this tutorial, we will learn how to perform batched graph classification
+with dgl via a toy example of distinguishing cycles from stars. From a synthetic
+dataset, we want to learn a binary classifier like below:
+"""
+
+###############################################################################
+#
+# .. image:: https://i.imgur.com/ZYaSoqO.png
```

Comment:
Maybe it's better to put the images into s3 buckets?

Reply:
Done.